src/AzureClient/Visualization/TargetStatusEncoders.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.Azure.Quantum.Client.Models;
+using Microsoft.Jupyter.Core;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Quantum.IQSharp.AzureClient
+{
+    internal static class TargetStatusExtensions
+    {
+        internal static Dictionary<string, object> ToDictionary(this TargetStatus target) =>
+            new Dictionary<string, object>()
+            {
+                { "targetName", target.Id },
+                { "currentAvailability", target.CurrentAvailability },
+                { "averageQueueTime", target.AverageQueueTime },
+                { "statusPage", target.StatusPage },
```

Comment:
Not sure we should expose status page here. I think that is more of an internal azure quantum field.

Reply:
Sure, I'll remove it. We can always add it back later if needed.